app/views/common/500.html.erb

```diff
@@ -1,21 +1,16 @@
 <%= alert :header => _("Oops, we're sorry but something went wrong"), :class => 'alert-danger',
   :actions => link_to(_('Back'), main_app.root_path, :class => 'btn btn-default'),
-  :text => h(exception.message),
+  :text => h(exception_message),
   :close => false %>
 
 <div class="pull-right">
   <%= documentation_button("7.3GettingHelp")%>
 </div>
+
 <p id="message">
   <%= _("If you feel this is an error with Foreman itself, please open a new issue with") %>
-  <%= link_to _("Foreman ticketing system"), "http://theforeman.org/projects/foreman/issues", :rel => "external" %>,
-  <%= _("You would probably need to attach the") %>
-  <%= link_to_function(_("Full trace"), "$('#backtrace').toggle()")%>
-  <%= _(", relevant log entries, and it is highly recommended to also attach the foreman-debug output.") %>
+  <%= link_to _("Foreman ticketing system"), "https://projects.theforeman.org/", :rel => "external" %>,
+  <%= _("Please include in your report the full error log that can be acquired by running: ") %>
+  <strong> foreman-rake errors:fetch_log request_id=<%= request_id %></strong>
```

Comment:
we can check if file logging is used by:
```
Foreman::Logging.config[:type] == "file"
```
So we could show this line in case it is `file` and in other case something like `Search your logs for request-id #{request_id`